### PR TITLE
handle SecurityError from localStorage

### PR DESCRIPTION
**Pull request by tuukka**
This happens if user has denied cookies in browser settings.

### CODE REVIEW

File: app/store/localStorage.js [hunk 1]
```diff
@@ -16,8 +19,18 @@ function setItem(key, value) {
 }
 
 function getItem(key) {
-  return (isBrowser && window.localStorage &&
-    window.localStorage.getItem(key)) || null;
+  if (isBrowser && window.localStorage) {
+    try {
+      return window.localStorage.getItem(key);
+    } catch (error) {
+      if (error.name === 'SecurityError') {
```
Comment (hannesj) [inline app/store/localStorage.js]: Could the catch block (or even the whole try/catch) be extracted, so it doesn't need to be duplicated?

File: app/store/localStorage.js [hunk 2]
```diff
@@ -1,5 +1,15 @@
 import { isBrowser, isWindowsPhone, isIOSApp } from '../util/browser';
 
+function handleSecurityError(error, logMessage) {
+  if (error.name === 'SecurityError') {
+    if (logMessage) {
+      console.log(logMessage); // eslint-disable-line no-console
+    }
```
Comment (hannesj) [inline app/store/localStorage.js]: Should we have else here?

Comment (tuukka) [inline app/store/localStorage.js]: No, since the only thing to do is to log any messages.

Comment (hannesj) [inline app/store/localStorage.js]: Uh, of course...

**Pull request merged**